vault/blog.community.2021.new-user-survey.md

```diff
@@ -0,0 +1,173 @@
+---
+id: AitIUzlK955P5EFojPjxz
+title: 2021 Dendron New User Survey Results
+desc: >-
+  The new user survey was open for several months in 2021, where we were prompting new Dendron users within VS Code itself. We had 2561 respondents take that opportunity to give us feedback.
+updated: 1642703269290
+created: 1642630788714
+image:
+  url: >-
+    https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-banner.png
+  alt: Computer sitting on a desk with plants
+date: '2022-01-20'
+excerpt: >-
+  The new user survey was open for several months in 2021, where we were prompting new Dendron users within VS Code itself. We had 2561 respondents take that opportunity to give us feedback.
+author: Derek Ardolf
+published: true
+---
+
+![Computer sitting on a desk with plants](https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-banner.png)
+
+The new user survey was open for several months in 2021, where we were prompting new Dendron users within VS Code itself. We had 2561 respondents take the opportunity to give us feedback.
+
+## What backgrounds do our users have?
+
+![Bar graph displaying different backgrounds of users](https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-primary-background.png)
+
+New users were only able to select a single, primary background. Just over half of respondents had a background as **Software Developers** (53%), while the next largest group were primarily **Students** (14%).
```

Comment:
heads up, this wasn't the case initially (users were able to select multiple). to clear up confusion, just say that users were able to select a background and drop primary distinction